flow/embedded_views.h

```diff
@@ -196,6 +196,8 @@ class ExternalViewEmbedder {
 
   virtual bool SubmitFrame(GrContext* context);
 
+  virtual bool MergeThreads() = 0;
```

Comment:
Maybe use a more generic name here, as this interface is platform agnostic and it might not be the case that this call actually leads to merging the threads on all platforms